starter-code/js/index.js

```diff
@@ -2,16 +2,25 @@ function deleteItem(e){
 
 }
 
-function getPriceByProduct(itemNode){
-
+// function getPriceByProduct(nodeItem)
+function getPriceByProduct(){
+  var priceItem = document.getElementsByClassName('.price-item');
```

Comment:
This is not working. You have two options:
Without dot (.)
```
document.getElementsByClassName('price-item');
```
or with dot and querySelector
```
document.querySelector('.price-item');
```